sdk/core/Azure.Core/src/Internal/StringExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Core
+{
+    internal static class StringExtensions
+    {
+        public static int IndexOfOrdinal(this string s, char c)
+        {
+#if NET5_0
+            return s.IndexOf(c, StringComparison.Ordinal);
+#else
+            return s.IndexOf(c);
+#endif
+        }
+
+        public static int GetHashCodeOrdinal(this string? s)
+        {
+            if (s == null) return 0;
+
+#if NET5_0
+            return s.GetHashCode(StringComparison.Ordinal);
+#else
+            return StringComparer.Ordinal.GetHashCode(s);
+#endif
+        }
+    }
+}
```

Comment:
Not a big deal, but if you're using VS our .editorconfig should've added a new line automatically. If using VSCode, consider installing the editorconfig.editorconfig extension to help enforce what few rules we have. This isn't one of them, but "EOF new lines" do help with diffs, i.e. when normalized to always have them vs. editors that add or remove them with each editor across machines/devs.

Reply:
I think rider doesn't do it automatically.